host/sgx/create.c

```diff
@@ -562,11 +572,119 @@ oe_result_t oe_sgx_validate_enclave_properties(
     return result;
 }
 
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+static oe_result_t _eeid_resign(
+    oe_sgx_load_context_t* context,
+    oe_sgx_enclave_properties_t* properties,
+    oe_eeid_t* eeid)
+{
+    oe_result_t result = OE_OK;
+
+    if (eeid && eeid->data_size > 0)
+    {
+        sgx_sigstruct_t* sigstruct = (sgx_sigstruct_t*)properties->sigstruct;
+
+        OE_SHA256 ext_mrenclave;
+        oe_sha256_final(&context->hash_context, &ext_mrenclave);
+
+        OE_CHECK(oe_sgx_sign_enclave(
+            &ext_mrenclave,
+            properties->config.attributes,
+            properties->config.product_id,
+            properties->config.security_version,
+            OE_DEBUG_SIGN_KEY, /* Use different key? */
```

Comment:
Can you elaborate on this question?

Reply:
This key is convenient because we don't need additional infrastructure to distribute it. I don't think there's a need for supporting other keys, but I can imagine that someone would ask for it at some point even if it's not necessary. I've removed the comment for now so as not to confuse anyone. @ad-l what do you think, is it necessary to support user-provided keys?